test_data/001_51806b39_1059de39_basic/added-01.err

```diff
@@ -0,0 +1,5 @@
+Error: PYLINT_WARNING:
+python/hello-world-python:1: C0114[missing-module-docstring]: Missing module docstring
+
+Error: PYLINT_WARNING:
+python/hello-world-python:4: C0116[missing-function-docstring]: api_method: Missing function or method docstring
```

Comment:
Shouldn't this file be a part of https://github.com/praiskup/vcs-diff-lint-testdata/tree/main ? It seems to be directly tied to https://github.com/praiskup/vcs-diff-lint-testdata/blob/main/python/hello-world-python

Reply:
This is a non-trivial question to answer; this is a kinda precedent (i never did a test suite like this one before).

The external repository is needed to "mimic" a real existing repository we want to test against, not really host the expected outputs (which are expected to change in the future, see that multiple possible outputs are accepted by test suite).

Said that;  the expected output file is _tight_ to a specific pair of SHA1 from the remote repo -> and these SHA1 checksums are not known before we do the commit;  it somewhat "feels" we should keep it here.